src/modules/auth/component/SignUp/SignUp.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+  Button, Grid,
+  TextField, Typography,
+} from "@mui/material";
+import { styled } from "@mui/material/styles";
+
+const TextFieldStyled = styled(TextField)({
```

Comment:
Why don't you use a custom component with this style? Name it as TextField and use MuiTextField inside of it. All the styles will be there too

Reply:
done